lib/tictactoe.rb

```diff
@@ -0,0 +1,85 @@
+require_relative './helper.rb'
+
+class TicTacToe
+  include Helperable
+  attr_accessor :board
```

Comment:
- [ ] Use `attr_readers/attr_writers` instead of `attr_accessors`. Using it will allow the `board` property to be changed from outside the class. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)). Someone can mess with this class which can introduce serious bugs.